src/typescript/typescript.ts

```diff
@@ -22,6 +23,78 @@ import {
 } from "../typescript";
 import { deepMerge } from "../util";
 
+/**
+ * @see https://kulshekhar.github.io/ts-jest/docs/getting-started/options
+ */
+export interface TsJestTransformOptions {
+  /**
+   * Custom TypeScript AST transformers
+   *
+   * @default auto
+   */
+  readonly astTransformers?: Record<string, any>;
+  /**
+   * Babel(Jest) related configuration.
+   *
+   * @default false
+   */
+  readonly babelConfig?: boolean | string | Record<string, any>;
+  /**
+   * TypeScript module to use as compiler.
+   *
+   * @default "typescript"
+   */
+  readonly compiler?: string;
+  /**
+   * Diagnostics related configuration.
+   *
+   * @default true
+   */
+  readonly diagnostics?: boolean | Record<string, any>;
+  /**
+   * Run ts-jest tests with this TSConfig isolatedModules setting.
+   *
+   * You'll lose type-checking ability and some features such as const enum, but in the case you plan on using Jest with the cache disabled (jest --no-cache), your tests will then run much faster.
+   * @see https://kulshekhar.github.io/ts-jest/docs/getting-started/options/isolatedModules
+   *
+   * @default false
+   */
+  readonly isolatedModules?: boolean;
+  /**
+   * Files which will become modules returning self content.
+   *
+   * @default disabled
+   */
+  readonly stringifyContentPathRegex?: string | RegExp;
```

Comment:
RegExp doesn't translate to other jsii languages. See failing build.
```suggestion
  readonly stringifyContentPathRegex?: string;
```